### ISSUE TITLE
Patches not working

### ISSUE THREAD

**Issue by illusion0001** [GoldHEN/GoldHEN_Patch_Repository#3]
Tracker for patches that are not working can be reported here.

You must provide the following details:

```
- Game: (game, version and ID)
- Patches enabled:
  - (list enabled patch here)
  - another line
- Where plugin was downloaded from:
- PS4 firmware version:
```

**Comment by OliverTauloski** [comment]
- Game: (Elden Ring version 1.05 and ID CUSA28863)
- Patches enabled:
  - 60 fps patch
  - 720p resolution patch
- Where plugin was downloaded from: https://mega.nz/file/VYQHQAYZ#Wq8yXzOV8gtrdTauwu1cGLU90eGjNjo5wG4gFoP9Q6w
- PS4 firmware version: 9.00
- Log file from PS4 at port 3232( I didn't understand this one, can you please explain how to get this one?)

**Comment by illusion0001** [comment]
Last one is optional. I fixed the patch and it should work now

**Comment by OliverTauloski** [comment]
Okay I will update the patches and I will try again and let you know

**Comment by OliverTauloski** [comment]
I tried it, It works fine, as expected around 50-60 fps. Thank you a lot.

**Comment by k3lee** [comment]
- Game: Bloodborne The Old Hunters Edition, v1.09 CUSA03023
- Patches Enabled: 60 FPS, Resolution Patch, Skip Intro
- Where plugin was downloaded from: https://github.com/GoldHEN/GoldHEN_Plugins_Repository/releases/download/1.155/GoldPlugins-1.155-be6666ee.zip
- PS4 firmware version: 9.00
- Log File: [klog.txt](https://github.com/GoldHEN/GoldHEN_Patch_Repository/files/10337169/klog.txt)

**Comment by illusion0001** [comment]
Bad or config file doesn't exist.

```
<118>[GoldHEN] <Plugins Loader> Started successfully
<118>[GoldHEN] <Plugins Loader> Config parser failed to parse config: /data/GoldHEN/plugins.ini
```

**Comment by k3lee** [comment]
Thank you very much! Missing config file was the reason plugins did not work.

**Comment by AbdelrahmanMahmoudMD** [comment]
- Game: (Elden Ring, 1.00, CUSA28863)
- Patches enabled: Any patch, however cheats are working
- Where plugin was downloaded from: official github, update button inside GoldHen cheat manager)
- PS4 firmware version: 9.00
- Log file from PS4 at port 3232 (optional) = don't know how to do this

**Comment by MrNabmas** [comment]
- Game: Horizon Forbidden West CUSA28651 1.18
- Patches enabled:
  - 60 fps
- Where plugin was downloaded from: Goldhen cheats manager 
- PS4 firmware version: 9:00 Base PS4
- Log file from PS4 at port 3232 (optional) ( tried to do this but not sure i used the ftp client also enabled klog in settings ) 
- 
- starting screen shows 60fps , when its loading after its like 42fps then goes to 30fps when in game playing
- also just wondering is it possible to do 40 fps patches? i heard the input lag is better, thanks
- 
- Game: Assassins creed 3 remastered and liberation remastered CUSA11560 1.03
- Patches enabled:
  - 60fps
- Where plugin was downloaded from: Goldhen cheats manager 
- PS4 firmware version: 9:00
- Log file from PS4 at port 3232 (optional)

**Comment by gorshco** [comment]
- Game: Crisis Core Final Fantasy 7 Reunion, CUSA31349, 1.03 + Backport (could not find a non-backported update)
- Patches enabled:
![20230211_135537_00002811](https://user-images.githubusercontent.com/35520491/218275954-2f283366-3cb7-43e6-88ed-efb5bd7180a4.jpg)
- PS4 firmware version: 9.00

(*) The plugin works fine and applies patches to other games I have installed

**Comment by illusion0001** [comment]
Have you update the plugins

**Comment by MrNabmas** [comment]
yep says successful for both cheats and plugins, only 2 that work for me is bloodborne and hellblade, thanks

**Comment by illusion0001** [comment]
What is the version of the Plugins

**Comment by MrNabmas** [comment]
elden ring 1.10 
horizon 1.18 

cant test these below as deleted them 
sonic 1.10
assassins creed liberation 1.03
assassins creed 1.03

**Comment by illusion0001** [comment]
That's not it

**Comment by MrNabmas** [comment]
where is the version number? had a look but couldn't see anything related

**Comment by illusion0001** [comment]
No easy way but opening the prx in notepad and search for 2022 or 2023. Version is build date and number

**Comment by MrNabmas** [comment]
(Release) Dec 25 2022 @ 00:04:24 (%s:%d)   %s: not found

i had to use notepadd ++ as notepad didnt find anything, shall i replace the files? i downloaded the plugins folder and ini that one is Jan 25 2023 @ 01:54:58

double checking you mean game patch prx

**Comment by illusion0001** [comment]
Yes replace it

**Comment by illusion0001** [comment]
Replace just the prx, nothing else

**Comment by MrNabmas** [comment]
did that, but no difference read the previous reply from someone else and had to add the plugin.ini to data/goldhen

I didn't get the patches applied notification when launching a game but now i do ( not sure if i missed that feature )

horizon forbidden west 35-65-18fps basically looks very choppy like its skipping/missing frames 

all work below 
Elden ring works fine 3 patches applied 6 patch lines applied 

Hellblade went from 45fps to locked 60fps even though i thought it was working beforehand

bloodborne is locked 60 now too, thanks for the help!

**Comment by Count-MHM** [comment]
- Game:
  - Name : The Last of Us Part 2
  - Title ID : CUSA10249
  - Version : 1.09
- Patches enabled:
  - 60 FPS
  - 1080P for PRO
- Where plugin was downloaded from:
  - https://github.com/GoldHEN/GoldHEN_Plugins_Repository/releases/tag/1.177
- PS4 firmware version:
  - 6.72

**Comment by illusion0001** [comment]
What is your cheats manager version? @Count-MHM

**Comment by Count-MHM** [comment]
@illusion0001 1.03

**Comment by illusion0001** [comment]
How did you install the plugins @Count-MHM

**Comment by Count-MHM** [comment]
@illusion0001 copied them with FTP and enabled them from GoldHEN settings.

Just to make sure the plugin was loaded I added a game-specific setting to plugins.ini and it worked! But why it didn't work when it was listed under the default section?

Thanks!

**Comment by illusion0001** [comment]
show ini file

**Comment by illusion0001** [comment]
@Count-MHM show ini file

**Comment by Count-MHM** [comment]
@illusion0001 
```
; Note: lines starting with semicolon are for comments.
; Load plugins for any title.

[default]
/data/GoldHEN/plugins/game_patch.prx

; The Last of Us Part 2
[CUSA10249]
/data/GoldHEN/plugins/game_patch.prx

; Load plugins only for Playroom.
[CUSA00001]
/data/GoldHEN/plugins/afr.prx
/data/GoldHEN/plugins/no_share_watermark.prx
```

**Comment by illusion0001** [comment]
@Count-MHM works fine for me. reduced it down to just two lines
```ini
; Note: lines starting with semicolon are for comments.
; Load plugins for any title.

[default]
/data/GoldHEN/plugins/game_patch.prx
```

**Comment by 2DSamurai** [comment]
- Game: Wo Long Fallen Dynasty, v1.04 and CUSA36986
- Patches enabled:
  - 60 FPS
- Plugin was downloaded from: https://github.com/GoldHEN/GoldHEN_Plugins_Repository/releases/tag/1.177
- GoldHEN Cheats Manager version: 1.0.3
- PS4 firmware version: 9.00 base ps4
- Issue: Character movement/animation speed is slow, almost half of normal, especially in areas where FPS is 40-50fps. But in tutorial area it's ok (normal animation speed & 60fps). In any other level, movement/animation speed is bugged. Disabling patch reverts it back to normal speed, despite 30fps.

**Comment by ZupremeMaster** [comment]
At firsts patches did not work. After a guy on reddit suggested i deleted goldhen cheat app and deleted all the goldhen folders from data and insstalled back on now they are working. Something from older files or installation conflicting with the plugins perhaps ?

**Comment by illusion0001** [comment]
> Game name : Uncharted The Nathan Drake Collection Title ID : CUSA02344 Version : 1.02 Patches enabled: Enable Dev Menu Where plugin was downloaded from: https://github.com/GoldHEN/GoldHEN_Plugins_Repository/releases/tag/1.177 GoldHEN Cheats Manager version: 1.0.3 PS4 firmware version: 6.72
> 
> Sidenote: patches for Uncharted: Drake's Fortune & Uncharted 2: Among Thieves from the same Collection worked flawlessly. For Drake's Deception notification of applied patch strings appears on game start, but not signs of dev menu in the game itself.

Fixed in #40

**Comment by jackfavvv** [comment]
- Game: Fatal Frame: Mask of The Lunar Eclipse - [CUSA-36876] [v1.04] 
- Patches enabled: 
  - 60 FPS
- Where plugin was downloaded from:https://github.com/GoldHEN/GoldHEN_Plugins_Repository/releases/tag/1.188
- GoldHEN Cheats Manager version: 1.0.3
- PS4 firmware version: 6.72

**Comment by BillaBong1KP** [comment]
Game : Grand Theft Auto V {CUSA 00411} {1.46} 
Patches Enabled : 60 fps 
Where plugin was downloaded from:https://github.com/GoldHEN/GoldHEN_Plugins_Repository/releases/tag/1.188
GoldHEN Cheats Manager version: 1.1.0
PS4 firmware version: 9.00
UPDATE: Tried everything updated the plugins but nothing works although other patches are working .

**Comment by illusion0001** [comment]
> Game : Grand Theft Auto V {CUSA 00411} {1.46} Patches Enabled : 60 fps Where plugin was downloaded from:https://github.com/GoldHEN/GoldHEN_Plugins_Repository/releases/tag/1.188 GoldHEN Cheats Manager version: 1.1.0 PS4 firmware version: 9.00 UPDATE: Tried everything updated the plugins but nothing works although other patches are working .

@BillaBong1KP update database and try now

**Comment by BillaBong1KP** [comment]
> > Game : Grand Theft Auto V {CUSA 00411} {1.46} Patches Enabled : 60 fps Where plugin was downloaded from:https://github.com/GoldHEN/GoldHEN_Plugins_Repository/releases/tag/1.188 GoldHEN Cheats Manager version: 1.1.0 PS4 firmware version: 9.00 UPDATE: Tried everything updated the plugins but nothing works although other patches are working .
> 
> @BillaBong1KP update database and try now

Yeah , its working now . Thank You 😊

**Comment by illusion0001** [comment]
It's probably limited by the hardware as usual. PS4 cpu is not that powerful

**Comment by jackfavvv** [comment]
- Game: Fatal Frame: Mask of The Lunar Eclipse - [CUSA-36876] [v1.04] 
- Patches enabled: 
  - 60 FPS
  - No Film Grain
- Where plugin was downloaded from:https://github.com/GoldHEN/GoldHEN_Plugins_Repository/releases/tag/1.188
- GoldHEN Cheats Manager version: 1.1.0
- PS4 firmware version: 6.72

The character moving at half the speed resulting from 60 fps dipping down to 30fps
https://youtu.be/EKuMYVU37Vc

**Comment by BillaBong1KP** [comment]
> It's probably limited by the hardware as usual. PS4 cpu is not that powerful

Yeah , i am using Slim i think thats why i am not getting full fps .

**Comment by VideoBeeYT** [comment]
- Game: Stray, v1.03 CUSA24899
- 
- Patches enabled:
  - 60 FPS
  - 
- Where plugin was downloaded from: Goldhen cheats manager
- 
- GoldHEN Cheats Manager version: v01.10
- PS4 firmware version: 9.00

**Comment by qnblackcat** [comment]
- Game: Elden Ring, v1.15, CUSA28863
- Patches enabled:
  - 30 fps lock
  - 1080p for Pro
- Where plugin was downloaded from: Goldhen cheats manager/Github
- GoldHEN Cheats Manager version: 1.10
- PS4 firmware version: 9.00

**Comment by ianyanian** [comment]
- Game: Resident Evil 4 Remake, 1.10, CUSA33388
- Patches enabled:
  - Resolution Scale 67%
- Where plugin was downloaded from: GoldHEN Cheats Manager
- GoldHEN Cheats Manager version: 1.14
- PS4 firmware version: 9.00

Somehow resolution scale patches (any%) not working at all. The 30 FPS patch works

**Comment by Freefire666** [comment]
Greetings.

Game: Resident Evil 4 Remake. 1.10 CUSA33388
Patches enabled: All Resolution Scale Patches testet and not working
Where plugin was downloaded from: PS4 Cheats Manager v1.2.0 online Update
PS4 firmware version: 9.00

All Resolution Patches not working
30FPS patch works for me

**Comment by ViniciusBCarvalho** [comment]
- Game: (Horizon Zero Dawn, v1.54 and CUSA01967)
- Patches enabled:
  - 60 fps
- Where plugin was downloaded from: PS4CheatManager (from Internet)
- GoldHEN Cheats Manager version: v2.4b17.3
- PS4 firmware version: 11.00

30 fps in-game ... 60 fps in the menu. Let me know how I can help you!

@illusion0001 Thanks for everything!

**Comment by AlviseRecon** [comment]
- Game: Elden Ring, v1.13, CUSA18723
- Patches enabled:
  - Resolution Patch (900p for Base)
- Where plugin was downloaded from: Goldhen cheats manager/Github (2024-12-26 18:18:13 Thursday (UTC +0000)
- GoldHEN Cheats Manager version: 1.2.2
- PS4 firmware version: 9.00

The "Resolution Patch (720p for Base)" works perfectly, the 900p patch crashes the game at startup, right after the "patch applied" popup goes out (ERROR CB-34878-0).

Thanks @illusion0001 for hard work!

**Comment by tabnk** [comment]
Tested on PS5 5.10. Sonic Frontiers v1.41 60fps patch not working.

**Comment by PawiX25** [comment]
*   **Game:** `God of War` (`v1.33`, `CUSA07412`)
*   **Patches enabled:**
    *   `720p Resolution`
    *  `60 FPS Unlock`
*   **Where patch file was downloaded from:** `GoldHEN Cheats Manager`
*   **GoldHEN Cheats Manager version:** `1.14`
*   **PS4 firmware version:** `9.60`

Enabling the 720p patch causes severe visual artifacts/distortion on the main menu, primarily affecting the left side of the screen (see attached image).

![Image](https://github.com/user-attachments/assets/67e4b774-5004-4d8b-abc9-d7d4e5839f26)

**Comment by Eduardo1911** [comment]
- Game: Horizon Forbidden West 1.18 CUSA28561
- Patches enabled:
  - both the 60fps and the skip intro one
- Where plugin was downloaded from: from right here, manually download them and transferred via usb
- GoldHEN Cheats Manager version: i am using itemflowz ver 1.10
- PS4 firmware version: on PS5 firmware 7.01

They actually do not load, i tried translating from mask to bytes using chatgpt but did not manage to get it to work.

The old versions of the patch work(they show patch applied) but are buggy, the 60fps one is stuttering.
